### PR TITLE
Transformers

### PR DESCRIPTION
**Pull request by meghanabhange**
Adding code for pretraining LM and fine-tuning classifier for 
- BERT
- DistilBERT
- RoBERTa

### CODE REVIEW

File: HinglishBERTFinetuning.ipynb [hunk 1]
```diff
@@ -42,7 +42,7 @@
         "!pip install transformers\n",
         "!git clone https://github.com/NirantK/Hinglish.git\n",
         "%cd Hinglish\n",
-        "!git checkout all_berts"
+        "!git checkout transformers"
```
Comment (NirantK) [inline HinglishBERTFinetuning.ipynb:45]: Should this branch change be in the `master` branch?

File: HinglishDistilBERTMultilingual.ipynb [hunk 2]
```diff
@@ -42,7 +42,7 @@
         "!pip install transformers\n",
         "!git clone https://github.com/NirantK/Hinglish.git\n",
         "%cd Hinglish\n",
-        "!git checkout all_berts"
+        "!git checkout transformers"
```
Comment (NirantK) [inline HinglishDistilBERTMultilingual.ipynb:45]: Should this branch change be in the `master` branch?

Comment (meghanabhange) [reply HinglishDistilBERTMultilingual.ipynb]: Resolved by 4265454

File: HinglishBERTFinetuning.ipynb (hunk 1 above)
Comment (meghanabhange) [reply HinglishBERTFinetuning.ipynb]: Resolved by 4265454

File: HinglishDistilBERTMultilingual.ipynb [hunk 3]
```diff
@@ -0,0 +1,6788 @@
+{
```
Comment (NirantK) [inline HinglishDistilBERTMultilingual.ipynb:1]: <p>Use <code>pathlib</code>!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

File: HinglishRoBERTaBase.ipynb [hunk 4]
```diff
@@ -0,0 +1,2429 @@
+{
```
Comment (NirantK) [inline HinglishRoBERTaBase.ipynb:1]: <p>There's so much inline output across the files, I'm just gonna assume everything is A-okay kid. I cannot review the code on this interface. It's a bit too inconvenient.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (NirantK) [inline HinglishRoBERTaBase.ipynb:1]: <p>Hygiene habit: Add <code>index=False</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

**Pull request merged**